src/components/pnp/progress-sheet.ts

```
@@ -48,6 +49,18 @@ export abstract class ProgressSheet extends Sheet {
     return this.namedRange('PrcntFinishedYears');
   }
 
+  get reportingQuarter() {
+    const rptQtr = Number(this.namedRange('RptQtr').start.asString?.slice(1));
+    if (rptQtr < 1 || rptQtr > 4 || isNaN(rptQtr)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)